db/db_iter.cc

```diff
@@ -613,15 +626,17 @@ bool DBIter::ReverseToBackward() {
   return FindUserKeyBeforeSavedKey();
 }
 
-void DBIter::PrevInternal() {
+void DBIter::PrevInternal(const Slice* prefix) {
```

Comment:
Naming-wise, I wonder if prefix_ptr would be better than prefix. prefix_ already refers to Slice object and changing the type to Slice pointer in prefix might be confusing.

Reply:
I believe we have been following Google C++ Style, whose guideline is that type information should not be a part of the variable name, as using modern IDE can easily tell use type. But I just checked and didn't find it there. But I think the rule has been following in the whole code base and there is no reason to change.